src/fulcro/incubator/ui/core.cljs

```diff
@@ -14,6 +15,29 @@
   "Wrap a react component with localized css support (like on dom/*)"
   [component]
   (fn localized-factory
+    ([]
```

Comment:
I think you got it backwards.  Localized means it should use the localized-dom, and not the plain dom :)

Reply:
ah heck... i sure did lol